.travis.yml

```diff
@@ -40,58 +41,68 @@ matrix:
 
         # Try MacOS X
         - os: osx
-          env: PYTHON_VERSION=2.7 SETUP_CMD='test' OPTIONAL_DEPS=true
+          env: PYTHON_VERSION=2.7 SETUP_CMD='test'
 
         # Check for sphinx doc build warnings - we do this first because it
-        # runs for a long time
+        # runs for a long time. WCSAxes is a dependency for the docs.
         - os: linux
-          env: PYTHON_VERSION=2.7 SETUP_CMD='build_sphinx -w'  OPTIONAL_DEPS=true
-          # OPTIONAL_DEPS needed because the plot_directive in sphinx needs them
+          env: PYTHON_VERSION=2.7 SETUP_CMD='build_sphinx -w'
+               PIP_DEPENDENCIES='beautifulsoup4 wcsaxes'
```

Comment:
Two comments here:

* ``beautifulsoup4`` can be a conda dependency
* We can do ``wcsaxes --no-deps`` and then it won't install astropy